salt/pillar/nsot.py

```diff
@@ -0,0 +1,181 @@
+# -*- coding: utf-8 -*-
+'''
+A module that adds data to the Pillar structure from the NSoT API.
+
+.. versionadded:: 2018.3.4
```

Comment:
This must be changed to `Neon`, as the develop branch will not go into the 2018.3 release cycle.